.github/workflows/prod.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           RETRIES=5
           until [ $RETRIES -eq "0" ]
           do
-            if curl -sS http://localhost:8002/healthcheck; then
+            if curl -sS http://localhost/healthcheck; then
```

Comment:
here you need to request to `:8000` as by default it utilizes `:80` and nginx is not running who can proxy it to 8000. Or maybe in this case it's better to start `nginx` and then send the request to `:80` and it should be OK